user_input.rb

```diff
@@ -0,0 +1,10 @@
+class Input
+  def user_input
+    puts "Enter product details:"
+    input = Array.new
+    while(product_details = gets.chomp) != 'exit'
+      input.push(product_details) 
+    end 
+    return input   
```

Comment:
remove **return** keyword from here.
check how to return values in ruby functions.